shared/common-adapters/clickable-box.desktop.js

```diff
@@ -31,11 +30,12 @@ const styleFlatButton = {
   transform: 'none',
   textAlign: 'left',
   height: undefined,
-  width: '100%',
```

Comment:
The clickable box acts very differently than mobile which is a shame. Trying to make this more similar. We actually don't use it that much on desktop so instead i set width: 100% in the components that use it